src/commands/txCommands/util/branchCheck.js

```diff
@@ -0,0 +1,12 @@
+const { gitBranch$ } = require('./gitHelpers');
+
+const branchCheck$ = gitBranch$.do(branchName => {
+  if (branchName === 'master') {
+    console.log(
+      'do not run this script on master. it will kill the master resource on Transifex.'
```

Comment:
still weird without context - should this be named `checkNotMaster$` or something? not a blocker